src/main/java/org/sagebionetworks/web/client/presenter/NewAccountPresenter.java

```diff
@@ -138,6 +131,9 @@ public void onSuccess(String sessionToken) {
 				view.setLoading(false);
 				//success, send to login place to continue login process (sign terms of use...)
 				view.showInfo(DisplayConstants.ACCOUNT_CREATED, "");
+				if (accountCreationToken.getEncodedMembershipInvtnSignedToken() != null) {
+					globalAppState.setLastPlace(new EmailInvitation(accountCreationToken.getEncodedMembershipInvtnSignedToken()));

```

Comment:
nifty

Reply:
This actually took me a bit to figure out haha.